lnbits/tasks.py

```diff
@@ -133,8 +133,14 @@ async def check_pending_payments():
     the backend and also to delete expired invoices. Incoming payments will be
     checked only once, outgoing pending payments will be checked regularly.
     """
+    sleep_time = 60 * 30  # 30 minutes
 
     while settings.lnbits_running:
+        funding_source = get_funding_source()
+        if funding_source.__class__.__name__ == "VoidWallet":
```

Comment:
As a side note:
 - maybe funding sources should have an `info()` method (defined at the `base.Wallet` level as abstract)
 - this method would return info about the funding source supported features. Eg:
```json
{
    "invoices_stream": true, // some funding sources do not implement `paid_invoices_stream()` 
    "real_backend": true // in this case only VoidWallet would have false
}
```